doc.go

```diff
@@ -147,9 +147,15 @@
 //      CheckOrigin: func(r *http.Request) bool { return true },
 //  }
 //
-// The deprecated Upgrade function does not enforce an origin policy. It's the
-// application's responsibility to check the Origin header before calling
-// Upgrade.
+// We recommend the upgrader.Upgrade method to perform an upgrade
```

Comment:
There's no need to recommend the Upgrader.Upgrade. It's the only way non-deprecated API for upgrading a connection.